Updating /cmds - Main

XilentLVP opened the pull request:
Adding /account in /cmds -> Main
Edit for #663

XilentLVP committed:
Updating /cmds - Main

Adding /account in /cmds -> Main

XilentLVP committed:
Update OnDialogResponse.pwn

Specifer commented:
Ah cool..

XilentLVP committed:
Updating /help - Banking

Updated /help - Banking, since bank '/account' was removed.

XilentLVP committed:
Merge branch 'master' into master

XilentLVP committed:
Update help.json

RussellLVP commented on pawn/Resources/Callbacks/Others/OnDialogResponse.pwn:
```diff
@@ -297,7 +297,11 @@ deprecated_OnDialogResponse(playerid, dialogid, response, listitem, inputtext[])
-                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/help\r\n/stats\r\n/nos\r\n/getstats\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+
+                case 0: ShowPlayerDialog(playerid, DIALOG_COMMANDS_MAIN, DIALOG_STYLE_MSGBOX, "Main commands", "/account\r\n/help\r\n/stats\r\n/nos\r\n/getstats\r\n/showmessage\r\n/world\r\n/my\r\n/minigames\r\n/jump\r\n/cd", "Back", "Exit");
+=======
```
Would you mind fixing the merge conflict? Lines 301 and 303 roughly do the same, but we should only have one! 😛

RussellLVP commented:
Ace, thank you for the contribution @XilentLVP!!! One comment on a merge conflict.

XilentLVP committed:
Update OnDialogResponse.pwn

If it's not fixed,you can do that. I've just wanted to help, but I suck at it rn :P

RussellLVP reviewed (approved):
Thank you! Approved.

RussellLVP committed:
Merge branch 'master' into master
